src/content/admin/cloud-credentials/gcp.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: GCP
+description: Configure GCP credentials
+sidebar_label: gcp
+id: gcp-cloud-credentials
+---
+import Image from "@theme/Image";
+
+
+This tutorial will guide you on how to configure GCP credentials for your Okteto instance. In this specific guide we'll request read only access to cloud storage but you can provide access to any other AWS resource by using a role with more/different permissions.
+
+It's recommended that you have the [Google Cloud CLI installed](https://cloud.google.com/sdk/docs/install) to follow this tutorial.
+
+AWS credentials are configured using [Worlkload Identity Federation](https://cloud.google.com/iam/docs/workload-identity-federation).
+
+
+## Step 1: Create the Workload Identity Pool
```

Comment:
Can we add an intro paragraph explaining the goal of the step and what's the "Workload Identity Pool" for?